packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -15,7 +15,7 @@ import { getFiles } from './__utils__/getFiles'
 
 const CURRENT_BINARIES_HASH = enginesVersion
 
-const FIXED_BINARIES_HASH = '0c2898954d761d1c92f304ff1b7917c601c2e3d8'
+const FIXED_BINARIES_HASH = '0cecbd5867319b25d3d5110c16c398af16082790'
```

Comment:
This needs updating so the new engines namings are available on S3 I guess?

Reply:
Yes, old static hash used to tests here doesn't have the new bucket so this need to be last. Last time this string was updated was during M1 binary integration.